paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEvent.kt

```diff
@@ -457,6 +457,17 @@ internal sealed class PaymentSheetEvent : AnalyticsEvent {
         )
     }
 
+    class CannotLaunchLinkAndLpms(
```

Comment:
Can we rename this? We can launch it properly, but we can't return properly when the return url is triggered.